defaults/main.yml

```diff
@@ -238,6 +238,3 @@ onadata_odk_token_length: 7
 
 # Authentication settings
 onadata_api_disable_digest_auth: False
-
-# Project & XForm visibility
-onadata_allow_public_datasets: False
```

Comment:
We usually [check](https://github.com/onaio/ansible-onadata/blob/da180637cc51202e997fd3c72664f95a143b8dc0/templates/onadata_checkout_path/onadata/preset/local_settings.py.j2#L563-L566) whether this variable is defined or not